models/project_board.go

```diff
@@ -187,34 +190,63 @@ func GetUncategorizedBoard(projectID int64) (*ProjectBoard, error) {
 	}, nil
 }
 
-// LoadIssues load issues assigned to this board
-func (b *ProjectBoard) LoadIssues() (IssueList, error) {
+// LoadProjectIssues load project issues assigned to this board
+func (b *ProjectBoard) LoadProjectIssues() ([]*ProjectIssue, error) {
 	var boardID int64
 	if !b.Default {
 		boardID = b.ID
 
 	} else {
 		// Issues without ProjectBoardID
-		boardID = -1
+		boardID = 0
 	}
-	issues, err := Issues(&IssuesOptions{
-		ProjectBoardID: boardID,
-		ProjectID:      b.ProjectID,
-	})
-	b.Issues = issues
-	return issues, err
+	var projectIssues []*ProjectIssue
+	var issues []*Issue
+	if err := x.Table("issue").
+		Cols("issue.id, issue.repo_id, issue.index, issue.poster_id,issue.original_author,issue.original_author_id,issue.name,issue.content,issue.milestone_id,issue.priority,issue.is_closed,issue.is_pull,issue.num_comments,issue.ref,issue.deadline_unix,issue.created_unix,issue.updated_unix,issue.closed_unix,issue.is_locked,issue.project_issue, project_issue.id as project_issue_id, project_issue.project_id as project_issue_project_id, project_issue.project_board_id as project_issue_project_board_id").
```

Comment:
This is a very bad idea. Every time we change Issue we'll have to change this code.

Can't you use ``Cols("`issue`.*, ...)``?

Reply:
that should be possible at least on postgres, mysql and mssql. Don't know about sqlite

Reply:
Cols("`issue`.*, ...") doesn't work with sqlite, xorm generates the following query: "SELECT `issue`.`*`,..." and sqlite returns "Result: no such column: issue.*". If there is a way to prevent xorm from adding `` around * it would work but I couldn't figure it out.

Reply:
did you try `` `issue`.*``?

Reply:
if this also does not work than it is definitely bug in xorm... 
cc: @lunny

Reply:
@lafriks yes I tried, still not working.